### ISSUE TITLE
Improvements for Commands

### ISSUE THREAD

**Issue by OsakiTsukiko** [coltonk9043/Aoba-MC-Hacked-Client#18]
I haven't looked much through the sourcecode, but there are a few things that stand out (apart from commands not working at all at the moment).

- Why use arrays instead of ArrayList
- Commands are not saved in history

**Comment by coltonk9043** [comment]
Commands seem to work fine for me at the moment? If you have any stack trace or messages, I will gladly look through it to try and fix it. 

The main reason I use arrays is because I don't expect any of the parameters to ever really change in values or size, and I wouldn't hugely utilize the features of an Arraylist. Honestly, I am not happy with the way the current command system works right now. It was a very rushed feature that I wanted out fast back when I originally started it years ago. Hopefully some refinement will come soon.

I will look into adding commands into the history. It should be fairly easy to add.

Thanks for all of your help recently! I appreciate it. :)

**Comment by OsakiTsukiko** [comment]
![image](https://github.com/coltonk9043/Aoba-MC-Hacked-Client/assets/104794884/7bd5ea7e-95d1-437c-a43b-07ef23fafaae)

```
[14:02:13] [Render thread/INFO] (Minecraft) [System] [CHAT] §5[Aoba] §fError occured whilst running command. Please try again.
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.aoba.cmd.commands.CmdHelp.command(CmdHelp.java:37)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.aoba.cmd.CommandManager.command(CommandManager.java:113)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.gui.screen.ChatScreen.handler$zjp000$aoba$onSendMessage(ChatScreen.java:520)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.gui.screen.ChatScreen.sendMessage(ChatScreen.java:244)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.gui.screen.ChatScreen.keyPressed(ChatScreen.java:102)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.Keyboard.method_1454(Keyboard.java:395)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen.java:409)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.Keyboard.onKey(Keyboard.java:391)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.Keyboard.method_22678(Keyboard.java:477)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor.java:102)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.Keyboard.method_22676(Keyboard.java:477)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at org.lwjgl.system.JNI.invokeV(Native Method)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3403)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:201)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:212)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.util.Window.swapBuffers(Window.java:288)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1241)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:802)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.minecraft.client.main.Main.main(Main.java:250)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
[14:02:13] [Render thread/INFO] (Minecraft) [STDERR]:   at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)
```

java version
```
openjdk version "17.0.7" 2023-04-18
OpenJDK Runtime Environment (build 17.0.7+7)
OpenJDK 64-Bit Server VM (build 17.0.7+7, mixed mode)
```

**Comment by coltonk9043** [comment]
Should be fixed in the latest commit which fixed a lot of command functionality! Once I test it more thoroughly, I will upload a new release.

**Comment by yinlinghunjiang** [comment]
Same to me, when i entered .aoba help, it thrown an error to the log
```
[09:05:37] [Render thread/INFO]: [System] [CHAT] §5[Aoba] §fError occured whilst running command. Please try again.
[09:05:37] [Render thread/INFO]: [STDERR]: java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.aoba.cmd.CommandManager.command(CommandManager.java:98)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_408.handler$zzf000$aoba$onSendMessage(class_408.java:517)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_408.method_44056(class_408.java)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_408.method_25404(class_408.java:102)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_1454(class_309.java:395)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_437.method_25412(class_437.java:409)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_1466(class_309.java:391)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_22678(class_309.java:477)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_1255.execute(class_1255.java:102)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_309.method_22676(class_309.java:477)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at org.lwjgl.system.JNI.invokeV(Native Method)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3403)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at com.mojang.blaze3d.systems.RenderSystem.pollEvents(RenderSystem.java:201)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at com.mojang.blaze3d.systems.RenderSystem.flipFrame(RenderSystem.java:212)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_1041.method_15998(class_1041.java:288)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1523(class_310.java:1241)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.class_310.method_1514(class_310.java:802)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.minecraft.client.main.Main.main(Main.java:250)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:60)
[09:05:37] [Render thread/INFO]: [STDERR]: 	at oolloo.jlw.Wrapper.main(Wrapper.java:53)
```

**Comment by OsakiTsukiko** [comment]
yes, i tested last commit and still had some issues but didn't have time to write about them.

**Comment by OsakiTsukiko** [comment]
running .aoba crashes the client

```
---- Minecraft Crash Report ----
// Everything's going to plan. No, really, that was supposed to happen.

Time: 2023-07-02 16:43:34
Description: keyPressed event handler

java.lang.ArrayIndexOutOfBoundsException: Index 1 out of bounds for length 1
	at net.aoba.cmd.CommandManager.command(CommandManager.java:117)
	at net.minecraft.client.gui.screen.ChatScreen.handler$zkk000$aoba$onSendMessage(ChatScreen.java:520)
	at net.minecraft.client.gui.screen.ChatScreen.sendMessage(ChatScreen.java:244)
	at net.minecraft.client.gui.screen.ChatScreen.keyPressed(ChatScreen.java:102)
	at net.minecraft.client.Keyboard.method_1454(Keyboard.java:395)
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen.java:409)
	at net.minecraft.client.Keyboard.onKey(Keyboard.java:391)
	at net.minecraft.client.Keyboard.method_22678(Keyboard.java:477)
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor.java:102)
	at net.minecraft.client.Keyboard.method_22676(Keyboard.java:477)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:237)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1244)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.aoba.cmd.CommandManager.command(CommandManager.java:117)
	at net.minecraft.client.gui.screen.ChatScreen.handler$zkk000$aoba$onSendMessage(ChatScreen.java:520)
	at net.minecraft.client.gui.screen.ChatScreen.sendMessage(ChatScreen.java:244)
	at net.minecraft.client.gui.screen.ChatScreen.keyPressed(ChatScreen.java:102)
	at net.minecraft.client.Keyboard.method_1454(Keyboard.java:395)
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen.java:409)
	at net.minecraft.client.Keyboard.onKey(Keyboard.java:391)
	at net.minecraft.client.Keyboard.method_22678(Keyboard.java:477)
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor.java:102)
	at net.minecraft.client.Keyboard.method_22676(Keyboard.java:477)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)

-- Affected screen --
Details:
	Screen name: net.minecraft.client.gui.screen.ChatScreen
Stacktrace:
	at net.minecraft.client.gui.screen.Screen.wrapScreenError(Screen.java:409)
	at net.minecraft.client.Keyboard.onKey(Keyboard.java:391)
	at net.minecraft.client.Keyboard.method_22678(Keyboard.java:477)
	at net.minecraft.util.thread.ThreadExecutor.execute(ThreadExecutor.java:102)
	at net.minecraft.client.Keyboard.method_22676(Keyboard.java:477)
	at org.lwjgl.glfw.GLFWKeyCallbackI.callback(GLFWKeyCallbackI.java:44)
	at org.lwjgl.system.JNI.invokeV(Native Method)
	at org.lwjgl.glfw.GLFW.glfwWaitEventsTimeout(GLFW.java:3474)
	at com.mojang.blaze3d.systems.RenderSystem.limitDisplayFPS(RenderSystem.java:237)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient.java:1244)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:802)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)

-- Affected level --
Details:
	All players: 1 total; [ClientPlayerEntity['Player983'/211, l='ClientLevel', x=5.86, y=76.00, z=27.13]]
	Chunk stats: 961, 576
	Level dimension: minecraft:overworld
	Level spawn location: World: (0,78,32), Section: (at 0,14,0 in 0,4,2; chunk contains blocks 0,-64,32 to 15,319,47), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,-64,0 to 511,319,511)
	Level time: 93 game time, 93 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.client.world.ClientWorld.addDetailsToCrashReport(ClientWorld.java:458)
	at net.minecraft.client.MinecraftClient.addDetailsToCrashReport(MinecraftClient.java:2406)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient.java:821)
	at net.minecraft.client.main.Main.main(Main.java:250)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:468)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric

-- System Details --
Details:
	Minecraft Version: 1.20.1
	Minecraft Version ID: 1.20.1
	Operating System: Linux (amd64) version 6.1.34-1-lts
	Java Version: 17.0.7, N/A
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 699349944 bytes (666 MiB) / 2063597568 bytes (1968 MiB) up to 3995074560 bytes (3810 MiB)
	CPUs: 16
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 7 6800H with Radeon Graphics
	Identifier: AuthenticAMD Family 25 Model 68 Stepping 1
	Microarchitecture: Zen 3
	Frequency (GHz): -0.00
	Number of physical packages: 1
	Number of physical CPUs: 8
	Number of logical CPUs: 16
	Graphics card #0 name: GA107M [GeForce RTX 3050 Mobile]
	Graphics card #0 vendor: NVIDIA Corporation (0x10de)
	Graphics card #0 VRAM (MB): 4128.00
	Graphics card #0 deviceId: 0x25a2
	Graphics card #0 versionInfo: unknown
	Graphics card #1 name: Rembrandt [Radeon 680M]
	Graphics card #1 vendor: Advanced Micro Devices, Inc. [AMD/ATI] (0x1002)
	Graphics card #1 VRAM (MB): 258.00
	Graphics card #1 deviceId: 0x1681
	Graphics card #1 versionInfo: unknown
	Virtual memory max (MB): 15808.42
	Virtual memory used (MB): 9281.24
	Swap memory total (MB): 8192.00
	Swap memory used (MB): 0.00
	JVM Flags: 0 total; 
	Fabric Mods: 
		aoba: Aoba Hacked Client 1.3A
		fabric-api: Fabric API 0.83.1+1.20.1
		fabric-api-base: Fabric API Base 0.4.29+b04edc7a77
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.34+4d8536c977
		fabric-biome-api-v1: Fabric Biome API (v1) 13.0.10+b3afc78b77
		fabric-block-api-v1: Fabric Block API (v1) 1.0.9+e022e5d177
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.39+b3afc78b77
		fabric-client-tags-api-v1: Fabric Client Tags 1.0.20+b3afc78b77
		fabric-command-api-v2: Fabric Command API (v2) 2.2.11+b3afc78b77
		fabric-content-registries-v0: Fabric Content Registries (v0) 4.0.7+b3afc78b77
		fabric-convention-tags-v1: Fabric Convention Tags 1.5.3+b3afc78b77
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.18+aeb40ebe77
		fabric-data-generation-api-v1: Fabric Data Generation API (v1) 12.1.11+b3afc78b77
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.51+b3afc78b77
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.21+b3afc78b77
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.6.0+b3afc78b77
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.38+b04edc7a77
		fabric-gametest-api-v1: Fabric Game Test API (v1) 1.2.10+b3afc78b77
		fabric-item-api-v1: Fabric Item API (v1) 2.1.26+b3afc78b77
		fabric-item-group-api-v1: Fabric Item Group API (v1) 4.0.7+b3afc78b77
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.36+fb8d95da77
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.20+b3afc78b77
		fabric-loot-api-v2: Fabric Loot API (v2) 1.1.37+b3afc78b77
		fabric-message-api-v1: Fabric Message API (v1) 5.1.6+b3afc78b77
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.47+b3afc78b77
		fabric-models-v0: Fabric Models (v0) 0.3.35+b3afc78b77
		fabric-networking-api-v1: Fabric Networking API (v1) 1.3.8+b3afc78b77
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 11.0.6+b3afc78b77
		fabric-particles-v1: Fabric Particles (v1) 1.0.28+b3afc78b77
		fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.18+b3afc78b77
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 2.2.6+b3afc78b77
		fabric-renderer-api-v1: Fabric Renderer API (v1) 3.0.1+b3afc78b77
		fabric-renderer-indigo: Fabric Renderer - Indigo 1.3.1+b3afc78b77
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.33+b3afc78b77
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.26+b3afc78b77
		fabric-rendering-v1: Fabric Rendering (v1) 3.0.6+b3afc78b77
		fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.3.5+ea08f9d877
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.11.7+f7923f6d77
		fabric-screen-api-v1: Fabric Screen API (v1) 2.0.6+b3afc78b77
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.27+b3afc78b77
		fabric-sound-api-v1: Fabric Sound API (v1) 1.0.12+b3afc78b77
		fabric-transfer-api-v1: Fabric Transfer API (v1) 3.2.2+b3afc78b77
		fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 4.2.0+b3afc78b77
		fabricloader: Fabric Loader 0.14.21
		java: OpenJDK 64-Bit Server VM 17
		minecraft: Minecraft 1.20.1
	Launched Version: Fabric
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: AMD Radeon Graphics (rembrandt, LLVM 15.0.7, DRM 3.49, 6.1.34-1-lts) GL version 4.6 (Core Profile) Mesa 23.1.2, AMD
	Window size: 1920x1011
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'; Server brand changed to 'fabric'
	Type: Integrated Server (map_client.txt)
	Graphics mode: fancy
	Resource Packs: fabric
	Current Language: en_us
	CPU: 16x AMD Ryzen 7 6800H with Radeon Graphics
	Server Running: true
	Player Count: 1 / 8; [ServerPlayerEntity['Player983'/211, l='ServerLevel[New World]', x=5.86, y=76.00, z=27.13]]
	Data Packs: vanilla, fabric
	Enabled Feature Flags: minecraft:vanilla
	World Generation: Stable
```

**Comment by OsakiTsukiko** [comment]
![image](https://github.com/coltonk9043/Aoba-MC-Hacked-Client/assets/104794884/a523e142-54dd-4ad2-a5e8-e6d6872193da)
🤨 🤨 🤨

**Comment by coltonk9043** [comment]
Ahahaha. I had to use UTF-8 for the formatting codes. I'll be releasing the fix as 1.3.2 tonight. Should fix most issues with commands.

**Comment by OsakiTsukiko** [comment]
![image](https://github.com/coltonk9043/Aoba-MC-Hacked-Client/assets/104794884/c7a2573c-be52-418f-b128-fbd5a5f92a68)

Attempted fix in #19

**Comment by OsakiTsukiko** [comment]
i guess everything has been resolved, closing the issue.
